idsvr/templates/deployment-admin.yaml

```diff
@@ -119,7 +142,7 @@ spec:
         - name: backup-config-script
           configMap:
             name: {{ include "curity.fullname" . }}-backup-config-script
-            defaultMode: 705
+            defaultMode: 0705
```

Comment:
allow read and execute for others? Why not 0700?

Reply:
Kubernetes mounts this file with root as owner and there is no way to change the owner currently. Just the permissions

Reply:
And the user running this script isn't root? Would 0701 be enough then?

Reply:
One would assume that 0701 should be enough but during a quick test on my local machine no backup was created after changing the permissions of the script to 0701.

Reply:
Very unexpected. Good that you tested, @ju-cu